package.json

```diff
@@ -143,7 +143,7 @@
     "typescript": "^4.9.5"
   },
   "peerDependencies": {
-    "@streamyard/spanner": "^7.3.0-2",
+    "@google-cloud/spanner": "^5.18.0",
```

Comment:
Shall this be 5.18 or 7.x?

Reply:
umm, for now let's leave it as it was so we have less conflicts when updating from typeorm

Reply:
But I see your point. Luckily, spanner nodejs library has not changed the interface since the 5.18 version. So we can use the 7.3.0 in our code without any issue.